lib/simulator-xcode-6.js

```diff
@@ -951,6 +951,68 @@ class SimulatorXcode6 extends EventEmitter {
     `);
   }
 
+  //region Keychains Interaction
```

Comment:
is this a standard thing or is it for a particular IDE?

Reply:
I assume nowadays it is supported by most of IDEs

Reply:
at least Eclipse, IDEA and VSCode